src/disco/themePreview.js

```diff
@@ -8,3 +8,7 @@ export default function themeAction(node, action, _doc = document) {
   event.initEvent(action, true, false);
   node.dispatchEvent(event);
 }
+
+export function getThemeData({ id, name, headerURL, footerURL, textcolor, accentcolor }) {
```

Comment:
This looks a bit odd, but looks like this will do the job of ensuring the shape of the data object?

Maybe a comment could be added in case someone looks at this and thinks it should be factored out.